src/Tribe/Plugins.php

```diff
@@ -39,7 +39,7 @@ class Tribe__Plugins {
 				'thickbox_url' => 'https://theeventscalendar.com/product/wordpress-events-calendar-pro/?TB_iframe=true',
 			],
 			[
-				'short_name'   => 'Community Events',
+				'short_name'   => 'Community',
```

Comment:
Does this need to be translated?

Reply:
No - first plugin names aren't typically translated (especially in the admin) and second it's _mostly_ used as a key (although it _is_ put into notices as well)